*** testsBoth/A-01-13-M-ok.out	2024-10-03 22:33:10.000000000 +0100
--- testsBoth/A-01-13-M-ok.outhyp	2024-10-04 19:54:42.027637373 +0100
***************
*** 44,50 ****
  10 - Consultas
  0 - Sair
  Escolha uma opção: 3
- Ficheiro sem nome. Guardar ficheiro como: ap01.dat
  Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
--- 44,49 ----
***************
*** 57,60 ****
--- 56,61 ----
  9 - Gestão de vacinas
  10 - Consultas
  0 - Sair
+ Escolha uma opção: ap01.dat
+ Número inválido!
  Escolha uma opção: 0
